packages/combo-box/src/vaadin-combo-box-scroller.js

```diff
@@ -40,6 +40,10 @@ export class ComboBoxScroller extends ComboBoxScrollerMixin(PolymerElement) {
           border-width: var(--_vaadin-combo-box-items-container-border-width);
           border-style: var(--_vaadin-combo-box-items-container-border-style);
           border-color: var(--_vaadin-combo-box-items-container-border-color, transparent);
+
+          /* Firefox returns empty styles when setting position in virtualizer-iron-list-adapter.js,
+             so we have set it explicitly here */
```

Comment:
The check in the virtualizer is mainly there to make sure that non-static positioning is used for the scroll container if the component author failed to add it to the component styles.

Ideally, all components using the virtualizer should set the proper positioning themselves and not rely on the virtualizer's fallback mechanism to set it, so I'd leave out this comment as it's not a workaround but the proper way to configure the scroll container.